hawkular-bus-common/src/main/java/org/hawkular/bus/common/consumer/BasicMessageListener.java

```diff
@@ -44,6 +44,11 @@ protected BasicMessageListener(Class<T> jsonDecoderRing) {
         super(jsonDecoderRing);
     }
 
+    protected BasicMessageListener(ClassLoader basicMessageClassLoader) {
+        super(basicMessageClassLoader);
+        // TODO Auto-generated constructor stub
```

Comment:
I think we can remove this TODO - unless there is something else we need to add to the implementation? I don't think there is, which is why I bring this up.